daemon/containerd/image_history.go

```diff
@@ -2,18 +2,14 @@ package containerd
 
 import (
 	"context"
-	"sort"
 
-	"github.com/containerd/containerd/images"
 	containerdimages "github.com/containerd/containerd/images"
```

Comment:
LOL. Again!

We _really_ need to look which linter we're missing that should check for this.